apps/examples/wifi_manager/wm_test/Kconfig

```diff
@@ -24,4 +24,10 @@ config WIFIMANAGER_TEST_TRIAL
 	---help---
 		Number of test trial
 
+config WIFIMANAGER_AP_LIST_ITEMS_COUNT
```

Comment:
please follow Kconfig rule. you should add examples_ prefix

Reply:
Sir, changes done as suggested. Please review.